File: arithmetic.cpp
#include<iostream>

int main(){
    float a;
    float b;
    std::cout << "Enter any two numbers: \n";

    std::cin >> a;
    std::cin >> b;

    int sum = a + b;
    int diff = a - b;
    int product = a*b;
    float div = a/b;

    std::cout<< "Sum is :"<<sum<<"\n";
    std::cout<< "Diff is :"<<diff<<"\n";
    std::cout<< "Product is :"<<product<<"\n";
    std::cout<< "Div is :"<<div<<"\n";

    return 0;
    
}

File: Arithmetic.java
package series;
import java.util.Scanner;

public class Arithmetic {

 
	public static void main(String[] args) {
		 
		float a,b;
		
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter any two numbers : ");
		a = sc.nextFloat();
		b = sc.nextFloat();

		float sum,diff,mul,div;
		
		sum = a + b;
		diff = a- b;
		mul = a*b;
		div = a/b;
		
		System.out.println("Sum of the numbers is : "+sum);
		System.out.println("Difference of the numbers is : "+diff);
		System.out.println("Multiplication of the numbers is : "+mul);
		System.out.println("Division of the numbers is : "+div);
	}

}
